### PR TITLE
Split edpm-values into edpm-nodeset-values and edpm-deployment-values

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: automation/vars/default.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ vas:
             osdpd edpm-deployment-pre-ceph --for condition=Ready
             --timeout=1500s
         values:
-          - name: edpm-values
+          - name: edpm-nodeset-values
             src_file: values.yaml
```
Comment (abays) [inline automation/vars/default.yaml:35]: This `values.yaml` was renamed I think

File: automation/vars/default.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ vas:
         values:
           - name: service-values
             src_file: service-values.yaml
-          - name: edpm-values-post-ceph
+          - name: edpm-nodeset-values-post-ceph
```
Comment (cjeanner) [inline automation/vars/default.yaml:52]: isn't it a deployment, meaning `edpm-deployment-values-post-ceph` instead?

Comment (fao89) [reply automation/vars/default.yaml:54]: actually there are some changes to nodeset, it add some ceph related variables:
https://github.com/fao89/architecture/blob/edpm_values/examples/va/hci/nodeset-values.yaml#L18-L23

Comment (cjeanner) [reply automation/vars/default.yaml:54]: hmmm in the framework, those data are set from the ceph.yml play/hook. but OK, let's keep it here. We don't edit it anyway from the framework afaik.

**Comment by fao89** [comment]
Actually, this won't help,
I need to change the kustomization files to grab only nodeset/deployment and also have a kustomiztion that combines both for not breaking

**Pull request closed without merging**